routemaster/conftest.py

```diff
@@ -118,6 +118,40 @@
             ),
         ],
     ),
+    'test_machine_2': StateMachine(
```

Comment:
I think it's worth a comment (or other name) here explaining that this is a machine specifically for testing racey behaviours and shouldn't generally be extended.

Reply:
5519dfbbead074a153061e432b52128bfcfe5dfd